subsys/lorawan/nvm/lorawan_nvm_settings.c

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) 2021 Intellinium <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <LoRaMac.h>
+#include <settings/settings.h>
+#include "lorawan_nvm.h"
+#include <logging/log.h>
+#if defined(CONFIG_LORAWAN_SE_HAS_OWN_NVM)
```

Comment:
Can you please make this commit as an independent PR? We should merge this one asap and backport to LTS.

Reply:
working on it